pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -826,6 +825,14 @@ type DatacenterSpecKubevirt struct {
 	InfraStorageClasses []KubeVirtInfraStorageClass `json:"infraStorageClasses,omitempty"`
 }
 
+type NamespacedMode struct {
+	// Enabled indicates whether the single namespace mode is enabled or not.
+	Enabled bool `json:"enabled,omitempty"`
+	// Name is the name of the namespace to be used, if not specified the default "kubevirt-workload" will be used.
+	// +kubebuilder:default=kubevirt-workload
+	Name string `json:"name,omitempty"`
```

Comment:
Can't re-open https://github.com/kubermatic/kubermatic/pull/13614/files#r1727134553 due to the policies but my suggestion also was to rename the variable `Name` to `Namespace`. Naming it as namespace makes it self-explanatory, "name" is a bit subjective and can be confusing.

Reply:
Oh, that's right